cli/cmd/top.go

```diff
@@ -149,11 +151,12 @@ func newCmdTop() *cobra.Command {
 		"Display requests with this :authority")
 	cmd.PersistentFlags().StringVar(&options.path, "path", options.path,
 		"Display requests with paths that start with this prefix")
+	cmd.PersistentFlags().BoolVar(&options.withSource, "with-source", options.withSource, "Include the source column")
```

Comment:
I find boolean flags that default to true to be awkward to use from a command line perspective, since they require a boolean value to disable. For instance, to suppress the source column, you would need to run:

```
linkerd top --with-source=false
```

Whereas if the flag were defaulted to false, then you could run:

```
linkerd top --hide-sources
```

Or whatever you'd choose to call it in that case.